### PR TITLE
Build agent docker image during CI

### PR DESCRIPTION
**Pull request by IgorGov**

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -40,17 +40,18 @@ agent: ## Build agent.
 	@ls -l agent/build
 
 agent-debug: ## Build agent for debug.
-	@(echo "building mizu agent for debug.." )
+	@(echo "uilding mizu agent for debug.." )
```
Comment (gadotroee) [inline Makefile:43]: mistake?

File: .github/workflows/release.yml [hunk 2]
```diff
@@ -82,91 +82,8 @@ jobs:
             GIT_BRANCH=${{ steps.version_parameters.outputs.branch }}
             COMMIT_HASH=${{ github.sha }}
 
-  gcp-registry:
```
Comment (gadotroee) [inline .github/workflows/release.yml:85]: where do we add the GCR?

Comment (IgorGov) [reply .github/workflows/release.yml:85]: lets discuss it f2f

File: Makefile (hunk 1 above)
Comment (IgorGov) [reply Makefile]: fixed

**Review by RoyUP9 — APPROVED** [review]

**Pull request merged**